src/main/java/rx/internal/operators/OperatorToObservableSortedList.java

```diff
@@ -100,13 +103,8 @@ public void onNext(T value) {
         child.setProducer(producer);
         return result;
     }
-    // raw because we want to support Object for this default
-    @SuppressWarnings("rawtypes")
-    private static Comparator DEFAULT_SORT_FUNCTION = new DefaultComparableFunction();
 
     private static class DefaultComparableFunction implements Comparator<Object> {
-        DefaultComparableFunction() {
```

Comment:
Removing this adds a synthetic accessor method. Hopefully PMD implements my check for these...